crmd/throttle.c

```diff
@@ -102,14 +83,16 @@ int throttle_num_cores(void)
  *       This will return NULL if the daemon is being run via valgrind.
```

Comment:
... or if `cib` is run under different user/group than `crmd` (unless the latter runs atypically as root)

?